### DISCUSSION TITLE
Mined blocks are orphaned as parent tipset has delayed blocks

### DISCUSSION THREAD

**Discussion by summerxue1992** [filecoin-project/lotus#6701] [Storage Provider]
In the recent period, the cluster always has some solitary block problems. The reason is that when the block is packaged, the hash of the parent block is different from that on the chain, which is caused by one missing tip. How to solve the problem of incomplete tip information of the same height during node synchronization?

**Comment by LexLuthr** [comment]
Hi @summerxue1992 , Do you still need help with this?
